deployment/mmw_stack.py

```diff
@@ -72,7 +72,7 @@ def main():
                                     parents=[common_parser])
     mmw_ami.add_argument('--machine-type', type=str,
                          choices=['mmw-app', 'mmw-tiler', 'mmw-worker',
-                                  'mmw-monitoring'],
+                                  'mmw-monitoring', 'mmw-app,mmw-worker'],
```

Comment:
Fixed in https://github.com/WikiWatershed/model-my-watershed/commit/a64a6cabd4556e058532c6fb0ee88a8a6281ae39.